tools/eMMC/functions.sh

```diff
@@ -346,28 +404,29 @@ copy_boot () {
   echo_broadcast "==> rsync: /boot/uboot/ -> ${tmp_boot_dir}"
   #FIXME: Why is it rsyncing /boot/uboot ? This is wrong
   #rsync -aAx /boot/uboot/ /tmp/boot/ --exclude={MLO,u-boot.img,uEnv.txt} || write_failure
-  rsync -avAx /boot/* ${tmp_boot_dir} --exclude={MLO,u-boot.img,uEnv.txt} || write_failure
+  rsync -aAxz --human-readable --info=name0,progress2 /boot/* ${tmp_boot_dir} --exclude={MLO,u-boot.img,uEnv.txt} || write_failure
```

Comment:
This breaks Debian Wheezy... The only real downstream image is the MachineKit, they haven't fully moved to Jessie yet..

```
Copying: Current rootfs to /dev/mmcblk1p1
----------------------------------------
==> rsync: / -> /tmp/rootfs
----------------------------------------
rsync: --info=name0,progress2: unknown option
rsync error: syntax or usage error (code 1) at main.c(1453) [client=3.0.9]
writing to [/dev/mmcblk1] failed...
==> Stopping Cylon LEDs ...
```

It's easy to grep, Debian Wheezy as rsync: 3.0.9-4, while jessie has 3.1.1-3.. (testing 3.1.2)

```
$ rsync --version | grep version | awk '{print $3}'
3.1.2
```